mineral-ui/mineral-ui
# [Site] Use mineral theme

### Description
- Remove `siteTheme` and all references to it
- Remove `styleReset` and all references to it
- Change all theme variables to `mineralTheme` variables
- Remove superfluous ThemeProvider around Nav
- Remove underlines from Links in Nav
- Fix bug with GitHub link in Footer

### Motivation and context

This provides a more realistic foundation upon which to further improve our Site. It also served as a proving ground for the ideas in #228.

### Screenshots or videos, if appropriate

<img width="640" alt="screen shot 2017-08-09 at 1 13 57 pm" src="https://user-images.githubusercontent.com/486540/29139608-bf2277dc-7d04-11e7-8957-50944d2c3b4e.png">

### How to test
1. `cd mineral-ui/packags/site && npm start`
1. Confirm things look right
1. For extra credit, go change the `font-size-base` value and confirm that none of the sizes change



### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Meh](https://media.giphy.com/media/UO95NWY0PmoWk/giphy.gif)
